server/src/main/java/org/apache/iotdb/db/engine/flush/MemTableFlushTask.java

```diff
@@ -170,94 +189,104 @@ private void writeOneSeries(TVList tvPairs, IChunkWriter seriesWriterImpl,
     @Override
     public void run() {
       long memSerializeTime = 0;
```

Comment:
You should use class variable `memSerializeTime` rather than this local variable

Reply:
Fixed~